packages/ui/src/lib/sidebar/Sidebar.stories.svelte

```diff
@@ -0,0 +1,225 @@
+				<p>Two stores accompany the sidebar component</p>
+				<ul class="-mt-2 ml-4 list-outside list-disc space-y-1">
+					<li><code>sidebarOpen</code></li>
+					<li><code>sidebarLayout</code></li>
+				</ul>
+			</SidebarHeader>
+
+			<SidebarSection title="$sidebarOpen">
+				<p>
+					<code>$sidebarOpen</code> is true if the sidebar is open and false if not.
+				</p>
+
+				<p>
+					Setting this store true or false will toggle the sidebar open or closed respectively. Any
+					reactive content using the store will receive the change and update accordingly.
+				</p>
+			</SidebarSection>
+
+			<SidebarSection title="$sidebarLayout">
+				<p>
+					<code>$sidebarLayout</code> is an object contains information about how the sidebar is laid
+					out with a few utility functions to ensure readable user code.
+				</p>
+
+				<p>
+					A new object is set each time the sidebar is resized so any content that needs to adapt
+					based on the sidebar layout can do so. It currently contains the following functions and
+					field with more functions likely to be added overtime:
+				</p>
+
+				<ul class="-mt-2 ml-4 list-outside list-disc space-y-1">
+					<li><code>isHorizontal()</code>: returns true if <code>mode</code> is 'horizontal'</li>
+					<li><code>isVertical()</code>: returns true if <code>mode</code> is 'vertical'</li>
+					<li><code>isRight()</code>: returns true if <code>alignment</code> is 'right'</li>
+					<li><code>isBottom()</code>: returns true if <code>alignment</code> is 'bottom'</li>
+					<li><code>mode</code>: either 'horizontal' or 'vertical'</li>
+					<li><code>alignment</code>: either 'right' or 'bottom'</li>
+					<li><code>hoodWidth</code>: sidebar hood width in pixels</li>
+					<li><code>hoodHeight</code>: sidebar hood height in pixels</li>
+					<li><code>contentWidth</code>: sidebar content width in pixels</li>
+					<li><code>contentHeight</code>: sidebar content height in pixels</li>
+				</ul>
+
+				<p>
+					User programmers should prioritise use of the functions over fields because they are less
+					error prone and provide better intentionality to readers.
+				</p>
+			</SidebarSection>
+		</SidebarContent>
+	</Sidebar>
+</Story>
+
+<Story name="The Sidebar Header">
+	<Sidebar>
+		<SidebarContent>
+			<SidebarHeader title="Sidebar header">
+				<p>
+					The <code>{'<SidebarHeader>'}</code> component should be the first one in the sidebar. It
+					uses the <code>{'<header>'}</code> tag.
+				</p>
+				<p>
+					The title is provided using the <i>title</i> property which may contain standard HTML tags,
+					e.i. not Svelte components.
+				</p>
+				<p>
+					Supply the description as unamed slotted content. Use a series of
+					<code>{'<p>'}</code> tags unless you need something bespoke.
+				</p>
+			</SidebarHeader>
+		</SidebarContent>
+	</Sidebar>
+</Story>
+
+<Story name="The Sidebar Section">
+	<Sidebar>
+		<SidebarContent>
+			<SidebarHeader title="Sidebar section">
+				<p>
```

Comment:
It would be useful to have an example of a "more info" modal here

```suggestion
			<SidebarHeader title="Sidebar section">
							<svelte:fragment slot="info">Here's some more info that goes with the header...</svelte:fragment>
				<p>
```